docs/30_commands.md

```diff
@@ -708,11 +708,23 @@ List Marketplace items
 
 #### Synopsis
 
-List the Marketplace items that the current user can access.
+List the Marketplace items that the current user can access.  
+This command allows you to view items in the Marketplace with various filters:
 
+*   Specify a `companyId` to view all Marketplace items for a specific company.
+*   Use the `--public` flag to list only the available public Marketplace items from any company.
+*   If both `--company-id` and `--public` are provided, the command will prioritize the `--public` flag, listing public Marketplace items and ignoring the `--company-id` flag.
```

Comment:
```suggestion
*   If both `--company-id` and `--public` are provided, the command will make the `--public` flag prevail, listing public Marketplace items and ignoring the `--company-id` flag.
```

Reply:
@pmarchini I would put this last recommendation not as a bullet point, but as an additional comment under the previous two, and introduce the sentence with something like "Be careful, if...."